File: src/java/Say.java
public class Say 
{
    public static void say(String m, String v) throws Exception {
        String cmd = "say -v " + v + " " + m;
        Runtime.getRuntime().exec(cmd).waitFor(); // wait until it finishes  
    }

    public static void main(String[] args) throws Exception {
        Say.say("Finish","Victoria");        // English
        Say.say("完了吧，如無意外", "Sin-ji"); // Cantonese
        Say.say("完結", "Ting-Ting");        // Chinese
        Say.say("終わり", "Kyoko");          // Japanese
        Say.say("종료", "Yuna");             // Korean
        Say.say("fin", "Monica");           // Spanish
        Say.say("finir", "Amelie");         // French        
    }
}

File: src/c_cpp/say.c
#include <stdio.h>
#include <stdlib.h>

void say(char *m, char *v) {
    char cmd[1000];
    sprintf(cmd, "say -v %s %s\n", v, m);
    system(cmd);
}

// u8 prefix: (UTF-8) encoding is needed for non-ASCII string 
int main() {
    say("Finish","Victoria");               // English
    say(u8"完了吧，如無意外", "Sin-ji");       // Cantonese
    say(u8"完結", "Ting-Ting");              // Chinese
    say(u8"終わり", "Kyoko" );               // Japanese
    say(u8"종료","Yuna");                    // Korean
    say("fin","Monica");                    // Spanish
    say("finir","Amelie");                  // French 
    return 0;
}